ansible/zapd_cron.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: zapd_cron
+  hosts: all 
+  become: yes
+  become_user: root
+
+  vars:
+   min_zap: 5000
+   max_zap: 10000
+
+
+  tasks:
+    - name: create the scripts directory
+      file:
+        path: /root/scripts
+        state: directory
+        mode: 0755
+        owner: root
+        group: root
+
+    - name: copy the zapd_balance.sh using the template module
+      template:
+        src: templates/zapd_balance.sh
+        dest: /root/scripts/zapd_balance.sh
+        mode: 0755
+   
+    - name : DEBUG
+      debug:
+        msg: "The host is {{ DEPLOY_HOST }}" 
+ 
+    - name: copy the zapd_block_height.sh using the template module
+      template:
+        src: templates/zapd_block_height.sh
+        dest: /root/scripts/zapd_block_height.sh
+        mode: 0755
+    
+    - name: copy the zapd_test.sh using the template module
```

Comment:
could we rename this to describe what its testing? zapd_isalive.sh or something?